### PR TITLE
Frontend Inheritance Organization (WIP) - Calendar Responsiveness Bug Fix

### PR DESCRIPTION
**Pull request by OdeChan**
Encountered a bug in responsiveness that makes the calendar not stretch till the bottom of the browser. 
Fixed using some flex/grid magic and changed the scroll listener to be on #calendar-grid instead of the of the whole window. 

Also rearranged some files and made better template bases.

Pushing so @aviadamar can continue. Hopefully didn't break anything 🤞

### CODE REVIEW

File: app/static/grid_style.css [hunk 1]
```diff
@@ -84,7 +78,7 @@ nav {
 #feature-settings {
     visibility: hidden;
     width: 0.1rem;
-    grid-row: 1/3;
+    /*grid-row: 1/3;*/
```
Comment (yammesicka) [inline app/static/grid_style.css:81]: Remove commented out code

File: app/static/grid_style.css [hunk 2]
```diff
@@ -116,9 +110,16 @@ header div {
...
+    flex: 1;
+    overflow-y: hidden;
 }
 
-.calendar {flex: 1;}
+.calendar {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+        height: 100%;
```
Comment (yammesicka) [inline app/static/grid_style.css:121]: Fix indentation

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: app/static/grid_style.css [hunk 3]
```diff
@@ -22,14 +9,15 @@ a {
 
 /* Grids */
 /* -> Website Structure */
-.container {
-    display: grid;
+.pyldr-container {
+    display: flex;
     grid-template-columns: 4.8rem minmax(0, auto) 1fr;
```
[the remark this answers is not in the document]
Comment (aviadamar) [reply app/static/grid_style.css:14]: Fixed.

File: app/static/grid_style.css (hunk 1 above)
Comment (aviadamar) [reply app/static/grid_style.css:81]: Fixed.

File: app/static/grid_style.css (hunk 2 above)
Comment (aviadamar) [reply app/static/grid_style.css:121]: Fixed.

**Pull request merged**